src/App.svelte

```diff
@@ -82,21 +86,33 @@
     <ColorPalette />
     <Header session={$session} {wallet} />
     <div class="wrapper">
-      <Router>
-        <Route path="/">
-          <Home />
-        </Route>
-        <Route path="vesting">
-          <Vesting {wallet} session={$session} />
-        </Route>
-        <Registrations {wallet} session={$session} />
-        <Seeds {wallet} session={$session} />
-        <Faucet {wallet} />
-        <Route path="/:addressOrName" let:params>
-          <Profile addressOrName={params.addressOrName} {wallet} />
-        </Route>
-        <Projects {wallet} />
-      </Router>
+      {#if $activeRouteStore.resource === "home"}
+        <Home />
+      {:else if $activeRouteStore.resource === "faucet"}
+        <Faucet {wallet} activeRoute={$activeRouteStore} />
+      {:else if $activeRouteStore.resource === "seeds"}
+        <Seeds
+          {wallet}
+          session={$session}
+          host={$activeRouteStore.params.host} />
+      {:else if $activeRouteStore.resource === "registrations"}
+        <Registrations
+          {wallet}
+          session={$session}
+          activeRoute={$activeRouteStore} />
+      {:else if $activeRouteStore.resource === "vesting"}
+        <Vesting {wallet} session={$session} />
+      {:else if $activeRouteStore.resource === "projects"}
+        <Projects {wallet} activeRoute={$activeRouteStore} />
+      {:else if $activeRouteStore.resource === "profile"}
+        <Profile
+          addressOrName={$activeRouteStore.params.addressOrName}
+          {wallet} />
+      {:else if $activeRouteStore.resource === "404"}
+        <NotFound title="404" subtitle="Nothing here" />
+      {:else}
+        {unreachable($activeRouteStore)}
+      {/if}
```

Comment:
Could we use a `switch` here to reduce the noise?

Reply:
It seems that a `switch` statement is eventually coming for svelte
https://github.com/sveltejs/svelte/issues/530

But maybe for now we can keep the `ifs` to avoid defining the component in a separate function, and the need to pass around the `wallet` props